### PR TITLE
Added registry.centos.org as defult registry

### PR DESCRIPTION
**Pull request by bamachrn**
* Images should be pulled in following sequence
1. internal registry
2. registry.centos.org
3. docker.io

### CODE REVIEW

File: provisions/roles/openshift/tasks/config.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@
   replace: >
     dest=/etc/sysconfig/docker
     regexp="^#*\s*ADD_REGISTRY=.*"
-    replace='ADD_REGISTRY="--add-registry {{ public_registry }}:5000"'
+    replace='ADD_REGISTRY="--add-registry {{ public_registry }}:5000 --add-registry registry.centos.org"'
```
Comment (navidshaikh) [inline provisions/roles/openshift/tasks/config.yml]: >Images should be pulled in following sequence
> internal registry
> registry.centos.org
> docker.io

so {{public_registry}} here is not really public registry but internal registry ?

**Comment by navidshaikh** [comment]
#dotests

File: provisions/roles/openshift/tasks/config.yml (hunk 1 above)
Comment (bamachrn) [inline provisions/roles/openshift/tasks/config.yml]: yes, it is the public registry only but without going through the nginx

Comment (navidshaikh) [inline provisions/roles/openshift/tasks/config.yml]: okay

File: provisions/roles/repo_tracking/tasks/main.yml [hunk 2]
```diff
@@ -94,7 +94,7 @@
 - name: Run postgres container
   docker_container:
       name: postgres
-      image: docker.io/centos/postgresql-95-centos7
+      image: registry.centos.org/sclo/postgresql-95-centos7
```
Comment (navidshaikh) [inline provisions/roles/repo_tracking/tasks/main.yml]: shouldn't this be a separate PR @rtnpro ?

Comment (bamachrn) [inline provisions/roles/repo_tracking/tasks/main.yml]: yes please

**Comment by bamachrn** [comment]
@rtnpro I intentionally did not change the name from public_registry to intranet_registry, as there will be one more entry in internal_registry. This will create more confusions. value referred by intranet_registry is actually the pubilc registry.

File: provisions/hosts.sample [hunk 3]
```diff
@@ -19,6 +19,7 @@ scanner_worker
 [all:vars]
 # update as needed
 public_registry=jenkins-slave
+intranet_registry=jenkins-slave:5000
```
Comment (bamachrn) [inline provisions/hosts.sample]: why we are keeping both here the public_registry and intranet_registry?

**Comment by navidshaikh** [comment]
> I intentionally did not change the name from public_registry to intranet_registry,

however the confusion stands ? IMO we change the names, we figure out the name when we want to add another one. For now, it would be good if we change the name.

**Comment by bamachrn** [comment]
#dotests
